### DISCUSSION TITLE
"Hide items with bad confidence" not able to be unticked

### DISCUSSION THREAD

**Discussion by phdfloppa** [Klocman/Bulk-Crap-Uninstaller#550] [Q&A] [answered]
How can I untick this option? It is greyed out:

![image](https://github.com/Klocman/Bulk-Crap-Uninstaller/assets/81775084/ec8e2e33-a70b-48cf-8599-c749c2a40765)

**Comment by Klocman** [comment, marked as answer]
If it's grayed out it means that there are no items with bad confidence.
